### PR TITLE
fix(Tooltip): Prevent callout content from gaining mouse focus

### LINKED ISSUE #12285: Tooltip cannot be interacted with when displayed on pressed button
### Environment Information

- **Package version(s)**: 7.83.0
- **Browser and OS versions**: Chrome Version 80.0.3987.132 (Official Build) (64-bit)/MacOS Mojave 10.14.6

### Please provide a reproduction of the bug in a codepen:

1. click on the [Click me first] button
    (due to mouse hover, a tooltip appears)
3. mouse over to the [Mouse over on me second] button in the tooltip
4. primary click the [Mouse over on me second] button

https://codepen.io/joshjcarrier/pen/mdJXGpK

#### Actual behavior:

The mouse click down on [Mouse over on me second?] button does not activate the button, and causes the tooltip to dismiss.

fabric_actual

#### Expected behavior:

The mouse click down on [Mouse over on me second?] button should activate the button, and prevent the tooltip from dismissing.

fabric_expected

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: n/a

### PR DESCRIPTION
**Pull request by joshjcarrier**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #12285
- [x] Include a change request file using `$ yarn change`

#### Description of changes

We've already concluded that [tooltips don't support focus](https://github.com/OfficeDev/office-ui-fabric-react/issues/10240#issuecomment-525906499) and in some scenarios allowing mouse down on tooltip content attempts to do so. This returns focus to the target element hosting the tooltip, and since the mouse is no longer hovering over it, the tooltip dismisses skipping any component being interacted with (like a link or button).

This adds a mouse down handler to the tooltip callout content to prevent focus from moving while still allowing elements under the cursor to receive the click.

#### Focus areas to test

* on the [Default Tooltip Usage](https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip) example, click the [Hover over me] button and then click the tooltip. Expect that it no longer dismisses unexpectedly.

* on the [Default Tooltip Usage](https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip) example, hover over the [Hover over me] button and double-click the text content `This is the tooltip content`. Expect that it no longer receives focus via text selection.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12344)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx [hunk 1]
```diff
@@ -45,6 +45,10 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
       maxWidth: maxWidth!
     });
 
+    const onMouseDown = (ev: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      ev.preventDefault();
```
Comment (aneeshack4) [inline packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx]: Looks like this has regressed a lot of Screener scenarios...
image
but the Screener results link itself only shows 2 that were changed in TeachingBubble this is a bit confusing... and they don't seem related to your change. I'm going to close and reopen to trigger another run.

**Comment by aneeshack4** [comment]
Pinging @micahgodbolt for codeowner approval

**Comment by micahgodbolt** [comment]
waiting for passing build

**Comment by aneeshack4** [comment]
@joshjcarrier Can you fix the build so that this can be merged?

**Comment by joshjcarrier** [comment]
Retriggering build

**Comment by ecraig12345** [comment]
I'm a little confused by this change. Seems like it has the side effect of disabling selection of tooltip text, which while it's not the primary thing you'd want to do with a tooltip, may be desirable in some cases (and I suspect someone will file an issue if we disable it).

**Comment by joshjcarrier** [comment]
Yes, I was comparing the options of:
1. tooltip contains focusable content, or
2. tooltip dismisses through matching interactions (e.g. [mouseIn/mouseOut](https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-21))

But with multiple issues (#11711 #11952 #10240) already citing tooltips that should not receive focus, I propose the inconsistent dismiss behavior (as reproed on the public docs) is the defect and focusable text (and the entailing accessibility concerns) should be handled with Callout/Dialog to be supported.
